src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -771,13 +771,18 @@ private List<PortViewModel> CreateProxyOutPorts(IEnumerable<PortModel> groupPort
                 .ToList();
 
             var newPortViewModels = new List<PortViewModel>();
-            for (int i = 0; i < groupPortModels.Count(); i++)
+            int index = 0;
+            foreach (var group in groupPortModels)
```

Comment:
Any specific reason to switch to this foreach loop? we already had index variable in the previous loop.